osu.Game/Screens/Play/Player.cs

```diff
@@ -145,10 +149,17 @@ protected override void OnEntering(GameMode last)
             base.OnEntering(last);
 
             (Background as BackgroundModeBeatmap)?.BlurTo(Vector2.Zero, 1000);
+            Background?.FadeTo((100f- dimLevel)/100,1000);
 
             Content.Alpha = 0;
         }
 
+        protected override bool OnExiting(GameMode next)
+        {
+            Background?.FadeTo(1f, 0);

```

Comment:
hmm, I found it weird if it still had the backgrouddim right after going back to songselect but I guess it would be better to transition gradually like the blurr does